webapp/src/components/formatMenu.tsx

```diff
@@ -38,7 +38,22 @@ const FormatMenu = (properties: FormatMenuProps) => {
         open={isMenuOpen}
         onClose={(): void => setMenuOpen(false)}
         anchorEl={anchorElement.current}
-        anchorOrigin={properties.anchorOrigin}
+        anchorOrigin={{
+          vertical: "bottom",
+          horizontal: "left",
+        }}
+        transformOrigin={{
+          vertical: "top",
+          horizontal: "left",
+        }}
+        sx={{
+          "& .MuiPaper-root": {
```

Comment:
We should try hard NOT to do this. This is an example of a _leaky abstraction_. We shouldn't care about the internals.

Also you could do a `StyledMenu` for it

Reply:
Thanks! Fixed in #183, I think.